### PR TITLE
SKN-81 pull channels from advertising platforms

### PR DESCRIPTION
**Pull request by PedroLabrador**

### CODE REVIEW

File: swagger.json [hunk 1]
```diff
@@ -296,7 +296,33 @@
                     }
                 }
             }
-        }
+        },
+        "/bq/channels": {
```
Comment (maacaro) [inline swagger.json]: @PedroLabrador this should be just /channels?

**Review by maacaro — CHANGES_REQUESTED** [review]
@PedroLabrador test are missing

File: swagger.json (hunk 1 above)
Comment (PedroLabrador) [reply swagger.json]: yes

**Review by maacaro — APPROVED** [review]
LGTM

**Pull request merged**